client/backwards_compatibility_test.go

```diff
@@ -278,7 +278,7 @@ func Test0Dot3RepoFormat(t *testing.T) {
 	delegations, err := repo.GetDelegationRoles()
 	require.NoError(t, err)
 	require.Len(t, delegations, 1)
-	require.Equal(t, "targets/releases", delegations[0].Name)
+	require.EqualValues(t, data.GUN("targets/releases"), delegations[0].Name)
```

Comment:
Should this be `data.Role` instead of `data.GUN`?  This wouldn't require using `EqualValues` then, and could just check the role against the delegation.